kolibri/core/assets/src/vue/core-modal/index.vue

```diff
@@ -73,13 +73,15 @@
         default: false,
       },
     },
-    attached() {
+    mounted() {
       this.lastFocus = document.activeElement;
-      this.focusModal();
-      window.addEventListener('focus', this.focusElementTest, true);
+      this.$nextTick(() => {
+        this.focusModal();
+        window.addEventListener('focus', this.focusElementTest, true);
```

Comment:
actually function  focusElementTest calls this.$refs.modal, so might be good to stay inside nextTick